src/components/AllAlerts/styles.module.css

```diff
@@ -22,79 +29,101 @@
 .flexCol {
   display: flex;
   flex-direction: row;
-  /* flex-wrap: nowrap !important; */
+  width: 90%;
 }
 
 .selct:focus {
   outline: none;
 }
 .selct {
+  border: none !important;
   outline: none !important;
 }
 
+.parentAlerts {
+  display: flex;
+}
 .upperRight {
   position: relative;
   flex-direction: row;
-  padding-left: 5px;
-  margin-top: 5px;
+  padding-left: 0.3125rem;
+  margin-top: 0.3125rem;
 }
 .upperLeft {
-  margin-top: 5px;
-  margin-left: -24px;
+  margin-top: 0.3125rem;
+  margin-left: -1.5rem;
 }
 .sevParagraph {
-  display: inline-flex;
   flex-wrap: nowrap;
   flex-direction: row;
-  font-size: 16px;
+  font-size: 1rem;
   margin-bottom: 0;
-  margin-top: -5px;
-  margin-left: 5px;
+  margin-top: -0.3125rem;
 }
 .red {
-  background-color: #f0190c;
+  text-indent: -624.9375rem;
+  width: 2.1875rem;
+  height: 1.5625rem;
+  background: url(./high.svg);
```

Comment:
I don't think I realized we needed this in two places, my bad! So that we only have one copy of the image, can you write this as `url(../AlertCard/high.svg)` and delete the ones in this folder?

Reply:
fixed